packs/st2/actions/workflows/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,37 @@
+---
+  chain:
+    -
+      name: "check_for_permission_to_send_anonymous_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::collect_anonymous_data"
+      on-success: "get_server_uuid"
```

Comment:
how about `on-failure` handler here to just print a nice output saying "Not sending anonimous data: call home is turned off". This way if user sees that despite him opting out, `call home` in the log, he can look and be sure that it 'not just failed", but was turned off.